src/main/java/com/wavefront/sdk/common/clients/WavefrontClient.java

```diff
@@ -388,6 +408,34 @@ public void sendDistribution(String name, List<Pair<Double, Integer>> centroids,
     }
   }
 
+  @Override
+  public void sendEvent(String name, long startMillis, long endMillis, @Nullable String source,
+                        @Nullable Map<String, String> tags,
+                        @Nullable Map<String, String> annotations)
+      throws IOException {
+    if (closed.get()) {
+      throw new IOException("attempt to send using closed sender");
+    }
+    String event;
+    URI uri = URI.create(this.clientId);
+    try {
+      if (uri.getScheme().equals(Constants.HTTP_PROXY_SCHEME)) {
```

Comment:
Is the assumption here that if scheme is HTTP it is proxy. HTTPS is direct ingestion?

Reply:
Yes. The path is formatted in https://github.com/wavefrontHQ/wavefront-sdk-java/blob/86ee8ba438101e104e4e3e15fb61d76e9a04cbc3/src/main/java/com/wavefront/sdk/common/clients/WavefrontClientFactory.java#L193. I will add the comment to clarify.